src/backend/distributed/sql/citus--9.5-1--10.0-1.sql

```diff
@@ -9,8 +9,44 @@ DROP FUNCTION IF EXISTS pg_catalog.citus_total_relation_size(regclass);
 #include "udfs/undistribute_table/10.0-1.sql"
 #include "udfs/create_citus_local_table/10.0-1.sql"
 #include "udfs/citus_set_coordinator_host/10.0-1.sql"
+#include "udfs/citus_add_node/10.0-1.sql"
+#include "udfs/citus_activate_node/10.0-1.sql"
+#include "udfs/citus_add_inactive_node/10.0-1.sql"
+#include "udfs/citus_add_secondary_node/10.0-1.sql"
+#include "udfs/citus_disable_node/10.0-1.sql"
+#include "udfs/citus_drain_node/10.0-1.sql"
+#include "udfs/citus_remove_node/10.0-1.sql"
+#include "udfs/citus_set_node_property/10.0-1.sql"
+#include "udfs/citus_unmark_object_distributed/10.0-1.sql"
+#include "udfs/citus_update_node/10.0-1.sql"
+#include "udfs/citus_update_shard_statistics/10.0-1.sql"
+#include "udfs/citus_update_table_statistics/10.0-1.sql"
+#include "udfs/citus_copy_shard_placement/10.0-1.sql"
+#include "udfs/citus_move_shard_placement/10.0-1.sql"
+#include "udfs/citus_drop_trigger/10.0-1.sql"
 
 #include "../../columnar/sql/columnar--9.5-1--10.0-1.sql"
 
 #include "udfs/time_partition_range/10.0-1.sql"
 #include "udfs/time_partitions/10.0-1.sql"
+
+ALTER FUNCTION master_conninfo_cache_invalidate()
+RENAME TO citus_conninfo_cache_invalidate;
```

Comment:
A slight downside of this approach is that it leaves the name "master_" deep down in the function.  See the last line:

```sql
\d pg_dist_partition
              Table "pg_catalog.pg_dist_partition"
┌──────────────┬──────────┬───────────┬──────────┬─────────────┐
│    Column    │   Type   │ Collation │ Nullable │   Default   │
├──────────────┼──────────┼───────────┼──────────┼─────────────┤
│ logicalrelid │ regclass │           │ not null │             │
│ partmethod   │ "char"   │           │ not null │             │
│ partkey      │ text     │           │          │             │
│ colocationid │ integer  │           │ not null │ 0           │
│ repmodel     │ "char"   │           │ not null │ 'c'::"char" │
└──────────────┴──────────┴───────────┴──────────┴─────────────┘
Indexes:
    "pg_dist_partition_colocationid_index" btree (colocationid)
    "pg_dist_partition_logical_relid_index" UNIQUE, btree (logicalrelid)
Triggers:
    dist_partition_cache_invalidate AFTER INSERT OR DELETE OR UPDATE ON pg_dist_partition FOR EACH ROW EXECUTE FUNCTION citus_dist_partition_cache_invalidate()

\sf citus_dist_partition_cache_invalidate()
CREATE OR REPLACE FUNCTION pg_catalog.citus_dist_partition_cache_invalidate()
 RETURNS trigger
 LANGUAGE c
AS '$libdir/citus', $function$master_dist_partition_cache_invalidate$function$
```

To mitigate that you'd still need to do a CREATE OR REPLACE FUNCTION before or after the rename. I'm also ok punting on that and doing further clean up in the future.